config.yml

```diff
@@ -648,6 +681,135 @@ production_factories:
     repair_inputs:
       Diamond:
         material: DIAMOND
+  Emerald_Cauldron:
+    name: Emerald Cauldron
+    fuel: 
+      Charcoal:
+        material: COAL
+        durability: 1
+    inputs:  
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 192   
+      Diamonds:
+        material: DIAMOND
+        amount: 128   
+      Iron:
+        material: IRON_INGOT
+        amount: 256   
+    recipes:
+      - Emerald_XP_0
+      - Emerald_XP_1
+      - Emerald_XP_2
+    repair_multiple: 10
+    repair_inputs:
+      Emeralds:
+        material: EMERALD
+        amount: 2
+      Diamonds:
+        material: DIAMOND
+        amount: 2  
+      Iron:
+        material: IRON_INGOT
+        amount: 25  

```

Comment:
for repair multiple of 10, to maintain 10% ratio, amount should be 2; unless you want it to cost install cost in Iron Ingots to repair.